openshift-install-powervs

```diff
@@ -1136,6 +1136,11 @@ function setup {
   success "setup command completed!"
 }
 
+function read_varfile {
+  arg=$1
+  grep "$arg" "$varfile" | xargs | grep -v "^#" | grep -v "^/" | awk '{print $3}' | sed 's/"//g'
```

Comment:
How about  `$(grep -v "^[#/]" $varfile | grep "$arg" $varfile | awk '{print $3}' | sed 's/"//g'` ?

Reply:
Have put `xargs` for trimming leading spaces.